### PR TITLE
JBPM-5444: Refactor for latest Lienzo Picture's constructor signatures.

### PR DESCRIPTION
**Pull request by romartin**
Depends on 
- https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/329
- https://github.com/uberfire/uberfire/pull/580

### CODE REVIEW

**Comment by romartin** [comment]
Hey @manstis , updated commit for 280-RELEASE. Thanks!

File: drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java/org/drools/workbench/screens/guided/dtree/client/widget/shapes/BaseGuidedDecisionTreeShape.java [hunk 1]
```diff
@@ -172,32 +172,27 @@ public void execute() {
...
-                selector.setAlpha( 0.01 );
-                selector.setLocation( new Point2D( offsetX,
-                                                   offsetY ) );
-                picture.setLocation( new Point2D( offsetX,
-                                                  offsetY ) );
-                controlGroup.add( picture );
-                controlGroup.add( selector );
-            }
-        } );
+        new Picture( resource, picture1 -> {
```
Comment (jomarko) [inline drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java/org/drools/workbench/screens/guided/dtree/client/widget/shapes/BaseGuidedDecisionTreeShape.java]: Hi @romartin 

The workbench build now fails with this message:
[ERROR] Errors in 'jar:file:/home/jomarko/.m2/repository/org/drools/drools-wb-guided-dtree-editor-client/7.0.0-SNAPSHOT/drools-wb-guided-dtree-editor-client-7.0.0-SNAPSHOT-sources.jar!/org/drools/workbench/screens/guided/dtree/client/widget/shapes/BaseGuidedDecisionTreeShape.java'
[INFO]          [ERROR] Line 175: The constructor Picture(ImageResource, (<no type> picture1) -> {}, boolean) is undefined
[INFO]          [ERROR] Line 175: The target type of this expression must be a functional interface


Please, could you investigate the problem? Thanks.

Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java/org/drools/workbench/screens/guided/dtree/client/widget/shapes/BaseGuidedDecisionTreeShape.java]: @jomarko Have you built ```droolsjbpm-build-bootstrap``` and ```uberfire``` from master too? Jenkins was playing up yesterday and it's quite possible the public SNAPSHOTs are old.

Comment (jomarko) [inline drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java/org/drools/workbench/screens/guided/dtree/client/widget/shapes/BaseGuidedDecisionTreeShape.java]: @manstis I am investigating. thanks for tip.

**Pull request merged**